services/wallet/service.go

```diff
@@ -48,6 +51,21 @@ func isReputationGeoEnabled() bool {
 	return toggle
 }
 
+// VerifiedWalletEnable enable verified wallet call
+var VerifiedWalletEnable = isVerifiedWalletEnable()
+
+func isVerifiedWalletEnable() bool {
+	var toggle = false
+	if os.Getenv("VERIFIED_WALLET_ENABLED") != "" {
```

Comment:
typically we shove configurations on the context, can you do a check in here if the configuration is on the context, and if so use that?

Reply:
It a copy from the code above :wink: will be removed the minute rep goes out which will be Wed